apps/bestofjs-nextjs/src/components/ui/command.tsx

```diff
@@ -28,7 +28,11 @@ Command.displayName = CommandPrimitive.displayName;
 const CommandDialog = ({ children, ...props }: DialogProps) => {
   return (
     <Dialog {...props}>
-      <DialogContent className="overflow-hidden p-0 shadow-2xl xl:max-w-[800px]">
+      <DialogContent
+        className="overflow-hidden p-0 shadow-2xl xl:max-w-[800px]"
+        aria-describedby={undefined}
```

Comment:
😨 
This is scary right?

Whats going on passing `undefined` to `aria-describedby`?

Reply:
Here is the exact warning message:

![image](https://github.com/user-attachments/assets/328604a2-b6da-48e8-87f8-7845998e0596)

I chose `aria-describedby={undefined}` because the Command palette already has this, for accessibility:

```
<DialogTitle className="sr-only">Search</DialogTitle>
```

Reply:
Oh, I learned something new today thanks.